packages/front-end/components/Metrics/MetricToolTipBody.module.scss

```diff
@@ -0,0 +1,22 @@
+@import "../../styles/colors.scss";
+
+.description {
```

Comment:
I would name this class `markdown` or `markdown-container` instead of `description` to keep the code a little more generic.